app/uk/gov/hmrc/eoricommoncomponent/frontend/views/enrolment_pending_against_group_id.scala.html

```diff
@@ -27,7 +27,7 @@
     otherService.map(other => ServiceName.longName(other)).getOrElse(messages("cds.enrolment.pending.otherService"))
 }
 
-@layout_di(messages("cds.enrolment.pending.title")) {
+@layout_di(messages("cds.enrolment.pending.title"), suppressTelephoneNumberDetection = false) {
```

Comment:
Turn this feature off on every page that displays the help-desk telephone number